kettle-plugins/kafka/src/main/java/org/pentaho/big/data/kettle/plugins/kafka/KafkaConsumerInputMeta.java

```diff
@@ -583,12 +596,22 @@ public void setMetastoreLocator( MetastoreLocator metastoreLocator ) {
     this.metastoreLocator = metastoreLocator;
   }
 
-  public void setAdvancedConfig( Map<String, String> config ) {
-    advancedConfig = config;
+  public void setConfig( Map<String, String> config ) {
+    this.config = config;
   }
 
-  public Map<String, String> getAdvancedConfig() {
-    return advancedConfig;
+  public Map<String, String> getConfig() {
```

Comment:
We should do this during the init phase instead of here in getConfig()

Reply:
see https://github.com/pentaho/pentaho-data-refinery/blob/master/src/main/java/org/pentaho/di/trans/steps/annotation/ModelAnnotationStep.java#L88

Reply:
@kurtwalker you mean in step's init method, right?

Reply:
yes

Reply:
Thanks for the link, will take a look tomorrow.